src/main/kotlin/app/nush/bot/commands/Nick.kt

```diff
@@ -0,0 +1,123 @@
+import app.nush.bot.commands.utils.Emojis.cross
+import app.nush.bot.commands.utils.Emojis.tick
+import app.nush.bot.commands.utils.dmUser
+import com.jessecorbett.diskord.api.rest.CreateDM
+import com.jessecorbett.diskord.dsl.Bot
+import com.jessecorbett.diskord.dsl.CommandSet
+import com.jessecorbett.diskord.dsl.command
+import com.jessecorbett.diskord.util.authorId
+import com.jessecorbett.diskord.util.changeNickname
+import com.jessecorbett.diskord.util.sendMessage
+import com.jessecorbett.diskord.util.words
+
+object Nick : Command {
+    lateinit var bot: Bot
+    override fun init(bot: Bot, prefix: CommandSet) {
+        this.bot = bot
+        with(prefix) {
+            command("nick") {
+                val newName: String = words.drop(1).joinToString(separator = " ")
+                println(newName)
+                if (newName != "") {
+                    request(
+                        bot,
+                        authorId,
+                        newName
+                    )
+                } else {
+                    dmUser(
+                        bot,
+                        authorId,
+                        "You may not request an empty nickname!"
+                    )
+                }
+            }
+        }
+        with(bot) {
+            reactionAdded { messageReaction ->
+                try {
+                    if (messageReaction.emoji.name != cross && messageReaction.emoji.name != tick) {
+                        return@reactionAdded
+                    }
+                    val guildClient = clientStore.guilds[config.guildId]
+                    val channelClient = clientStore.channels[config.excoChannelId]
+                    if (!guildClient.getMember(messageReaction.userId).user?.isBot!!) {
+                        val nickRequestMessage =
+                            channelClient.getMessage(messageReaction.messageId)
+                        if (nickRequestMessage.content.substring(
+                                nickRequestMessage.content.length - 96
+                            ) != "\". $tick if you want to accept the rename request, $cross if you do not want to accept the rename request"
+                        ) {
+                            return@reactionAdded
+                        }
+                        if (nickRequestMessage.authorId != botId) return@reactionAdded
+                        val newName: String =
+                            nickRequestMessage.content.subSequence(
+                                nickRequestMessage.content.indexOf("\"") + 1,
+                                nickRequestMessage.content.indexOf(
+                                    "\"",
+                                    nickRequestMessage.content.indexOf("\"") + 1
+                                )
+                            ).toString()
+                        if (messageReaction.emoji.name == cross) {
+                            clientStore.channels[clientStore.discord.createDM(
```

Comment:
The [`dmUser`](https://github.com/appventure-nush/appventure-bot/pull/7/files#diff-2c83c825a2f58f368981efef74d99186)  method has been added. You can replace all the `CreateDM`s with `dmUser`.